packages/plugin-baby/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "plugin-baby",
+  "module": "index.ts",
```

Comment:
Maybe also add the "main" field.

Reply:
I'd like to receive some education on best practices. That package only has ts export. I had some though time making our setup work. Not having `main` helped at some stage.

Reply:
I'll look into it I am not so familiar with what the current state of the module field is will report back.

Reply:
Here is the Proposal for modules: [here](https://github.com/dherman/defense-of-dot-js/blob/master/proposal.md)
Discussion about it on Stackoverflow: [here](https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for)

So the main field is intended to point to the file is getting loaded if you require the directory or package. 

Module is intended to point towards the es6 module version this.

If it really gives us a hard time to make it work with the main field then let's not do it.